FreeRTOS/FreeRTOS-Plus-TCP
# Fixing doxygen comments

Description
-----------
This PR updates the doxygen config file to the latest main branch config and fixes the doxygen comments.

**NOTE**: WIP. Do not merge yet.

Test Steps
-----------

`doxygen docs/doxygen/config.doxyfile 2>&1 | tee doxyoutput.txt`

Related Issue
-----------


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
